ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master__edge.yaml

```diff
@@ -189,6 +197,17 @@ tests:
         OPENSHIFT_VERSION=4.8
       PACKET_PLAN: m3.large.x86
     workflow: assisted-baremetal
+- as: e2e-ai-operator-ztp
+  optional: true
+  steps:
+    cluster_profile: packet-assisted
+    dependencies:
+      ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
+      INDEX_IMAGE: assisted-service-operator-index
+      OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
+    env:
+      INDEX_IMAGE_OVERRIDE: quay.io/edge-infrastructure/assisted-service-index:latest
```

Comment:
It maybe better to add it in the supplemental images, what do you think @osherdp

Reply:
that way, we just have to redefine `INDEX_IMAGE`

Reply:
and cleanup all references to `assisted-service-operator-index` as it is broken (promoted from Prow's `ci-index`)

Reply:
yeah it should be better than we have right now